### PR TITLE
Additional fixes found during review for remove/managed

### PR DESCRIPTION
**Pull request by thrix**
This PR fixes additional issues found during review:

- [x] Fix traceback
- [ ] Fix tests
- [ ] Add some basic unit test coverage

Note that it also brings in changes from current master, fell free to rebase and not pull them in.

### CODE REVIEW

File: tmt/steps/prepare/__init__.py [hunk 1]
```diff
@@ -65,10 +65,7 @@ def go(self):
     def run_how(self, how, what):
         """ Run specific HOW """
         self.info('Prepare', f"{how} = '{what}", 'yellow')
-        getattr(self,
-            f"how_{how}",
-            lambda: 'generic',
-            )(how, what)
+        getattr(self, f"how_{how}", self.how_generic)(how, what)
```
Comment (pvalena) [inline tmt/steps/prepare/__init__.py:68]: I'll merge 1bf8619e600e17d808430067d887b0e196dc210e, but maybe I'll squash later. I wonder why it's an issue though... it's just formatting change, right?

Comment (pvalena) [reply tmt/steps/prepare/__init__.py:68]: Oh I see the error now, sorry, you're right.

**Comment by pvalena** [comment]
Thanks!

**Comment by thrix** [comment]
@pvalena yw :) closing

**Pull request closed without merging**